crates/editor/src/display_map/inlay_map.rs

```diff
@@ -375,17 +377,17 @@ impl<'a> InlayBufferRows<'a> {
         self.transforms.seek(&inlay_point, Bias::Left, &());
 
         let mut buffer_point = self.transforms.start().1;
-        let buffer_row = if row == 0 {
+        let buffer_row = MultiBufferRow(if row == 0 {
             0
         } else {
             match self.transforms.item() {
                 Some(Transform::Isomorphic(_)) => {
                     buffer_point += inlay_point.0 - self.transforms.start().0 .0;
```

Comment:
This is kind of suspicious tbh

Reply:
Can you elaborate?

Reply:
I mean that it seems a bit suspicious how we intermix inlay points and multibuffer points ig

Reply:
That code seems fine, just quite specific: it tries to find an isomorphic transform boundary (in our case, end of the buffer text just before we insert the inlay hint), and uses the end of that buffer text as a start of the inlay row.

@\as-cii was the one who wrote it and knows it better, I trust him and our fuzzy tests that this works 🙂